python-avd/pyavd/_eos_designs/structured_config/metadata/__init__.py

```diff
@@ -37,6 +37,9 @@ def metadata(self) -> None:
         self._set_cv_pathfinder()
         if self.shared_utils.digital_twin:
             self._set_digital_twin()
+        validate_hardware_inventory = self.shared_utils.platform_settings.validate_hardware_inventory
+        if validate_hardware_inventory:
+            self.structured_config.metadata.hardware_requirements = validate_hardware_inventory
```

Comment:
I don't think you need to check for existence?

```suggestion
        self.structured_config.metadata.hardware_requirements = self.shared_utils.platform_settings.validate_hardware_inventory
```